sympy/utilities/tests/test_autowrap.py

```diff
@@ -226,3 +226,10 @@ def test_ufuncify_source():
 }
 #endif"""
     assert source == expected
+
+def test_issue_10274():
```

Comment:
I think the tests for these go in the external directory and they have decorators that ensure they only run if cython is installed.